### DISCUSSION TITLE
Where is the table filterDropdown close property coming from?

### DISCUSSION THREAD

**Discussion by igalci** [ant-design/ant-design#39174] [General]
### What problem does this feature solve?

https://ant.design/components/table#components-table-demo-custom-filter-panel

You will notice that the last parameter passed into the filterDropdown function is 'close', but when I try to replicate this on my end, 'close' is undefined, and the interface documentation also doesn't mention 'close' anywhere.

### What does the proposed API look like?

Where is it coming from?

Also, is there an documentation for the parameters passed into filterDropdown and how they work?
    filterDropdown: ({ setSelectedKeys, selectedKeys, confirm, clearFilters, close }) => ()

**Comment by ugshanyu** [comment]
Did you check your ant version? On version 3.0.0 there is not close function but there are setSelectedKeys, selectedKeys, confirm, clearFilters functions.

**Comment by coolcorexix** [comment]
I have this same problem, dont know why it works on the example
